src/js/index.js

```diff
@@ -0,0 +1,218 @@
+import CreatePage from './components/createHtml/createHtml';
+import * as board from './components/main/board';
+
+import '../css/main.scss';
+
+const createPage = new CreatePage();
+
+const LANG = { RU: 'ru', EN: 'en' };
+
+const state = {
+  lang: LANG.RU,
+  capsLock: false,
+};
+
+const KEYBOARD = {
+  tab: '  ',
+  space: ' ',
+  enter: '\n',
+};
+
+window.addEventListener('beforeunload', () => {
+  localStorage.setItem('language', state.lang);
+});
+
+window.addEventListener('DOMContentLoaded', () => {
+  const language = localStorage.getItem('language');
+  if (language !== null) {
```

Comment:
`if (!language) {`

or you can avoid if at all:

`state.lang = language || LANG.RU;`

If the second option suits you better, you can remove the default language assignment on the line 11:

`lang: '',`

Reply:
allready done